docs/bill_of_materials_orin.md

```diff
@@ -0,0 +1,74 @@
+# Bill of Materials - Jetson Orin Nano version
+
+This page lists all the parts you need to build the open-source JetBot with the new [**Jetson Orin Nano 8GB Developer Kit**](https://developer.nvidia.com/embedded/learn/get-started-jetson-orin-nano-devkit), along with purchasing links from popular vendors.
+
+Some of the parts are 3D printed.  We provide the STL files needed to print these parts.  Please see our [3d printing](3d_printing.md) page for tips on printing.
+
+### Common parts
+
+You need these components for each JetBot.
+
+|  **Part** | **Quantity** | **Cost** | **URL** | **Notes** |
+| --- | --: | --: | --- | --- |
+| Jetson Orin Nano 8GB Developer Kit | 1 |  $499.00 | [NVIDIA](https://store.nvidia.com/jetson/store/)  |  |
+| Micro SD card | 1 | $14.95 | [Amazon](https://a.co/d/0aOqPQh) | 128GB |
+| Battery | 1 | $12.99 | [Amazon](https://a.co/d/5DvsXGu) | PD 20W output, 10,000mAh |
+| USB-C PD cable | 1 | $10.99 | [Amazon](https://a.co/d/ihjA431) | USB-C to DC 5.5mm x 2.5mm, right angle |
```

Comment:
Not an issue, but do they have a smaller version of this cable?

Reply:
I have searched around and evaluated alternatives (such as shown in the photo below), but did not have a good minimum version of the cable.
I will test to see if I can cut and solder them internal cables to effectively shorten the cable, and if that is the case I will document for those people prefer the nice tight cable management.

![IMG_5918](https://github.com/NVIDIA-AI-IOT/jetbot/assets/25759564/934ac434-3656-4200-8411-e1c70fed51da)

Reply:
I think it's alright, was just wondering.  Soldering the internal cables is probably too much to ask developers.